seatunnel-engine/seatunnel-engine-storage/checkpoint-storage-api/src/main/java/org/apache/seatunnel/engine/checkpoint/storage/common/ProtoStuffSerializer.java

```diff
@@ -48,12 +43,13 @@ private static <T> Schema<T> getSchema(Class<T> clazz) {
     @Override
     public <T> byte[] serialize(T obj) {
         Class<T> clazz = (Class<T>) obj.getClass();
+        LinkedBuffer buffer = LinkedBuffer.allocate(LinkedBuffer.DEFAULT_BUFFER_SIZE);
```

Comment:
I don't quite understand the change here, could you please explain it?

Reply:
Just serialized buffers.he cannot share.